client/components/plans/plans-compare/index.jsx

```diff
@@ -233,6 +234,14 @@ const PlansCompare = React.createClass( {
 			const plans = this.getPlans();
 			const features = this.getFeatures();
 
+			abtest( 'wordadsInstantActivation' ) === 'enabled' && features.splice( 6, 0, {
```

Comment:
Should we also check for `config.isEnabled( 'manage/ads/wordads-instant' )`? Folks might be able to see copy, but won't be able to activate if they upgrade.